core/controllers/editor.py

```diff
@@ -573,11 +823,27 @@ class FetchIssuesHandler(EditorHandler):
     """
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'exp_version': {
```

Comment:
Any other rules for the version? Maybe >0?

Reply:
Added, please take a look.